articles/api/v1/use-cases.md

```diff
@@ -11,28 +17,28 @@ Currently, there are certain features and functionality that are only available
 
 ## Active Directory Connector Monitoring
 
-In APIv1, there is a `GET` endpoint that allows you to monitor the status of your Active Directory Connector:
+In API v1, there is a `GET` endpoint that allows you to monitor the status of your Active Directory Connector:
 
 `/api/connections/{AUTH0_CONNECTION}/socket`
 
```

Comment:
No, this is an undocumented endpoint (please see https://github.com/auth0/api2/issues/842 for more info).